runtime/balrt.h

```diff
@@ -121,6 +123,8 @@ typedef GC struct Mapping {
 typedef GC struct Error {
     TaggedPtr message;
     int64_t lineNumber;
+    uint32_t npcs;
```

Comment:
Not consistent with other names in this file

Reply:
Changed